eth/fetcher/block_fetcher.go

```diff
@@ -359,14 +359,6 @@ func (f *BlockFetcher) loop() {
 			}
 			// If too high up the chain or phase, continue later
 			number := op.number()
-			if number > height+1 {
```

Comment:
Thinking about it some more, I'm not sure this is a good change.. doesn't this logic ensure we fetch blocks monotonically? I imagine that's desirable, no?

Reply:
what do you mean by monotonically?

Reply:
I mean this code prevents us from sending blocks to InsertChain out of order. Lets merge for now, since it seems to address the problem, but please create an issue to revisit these downloader questions.